quickwit-serve/src/rest.rs

```diff
@@ -98,7 +107,7 @@ async fn search_endpoint<TSearchService: SearchService>(
     search_request: SearchRequestQueryString,
     search_service: &TSearchService,
 ) -> Result<SearchResultJson, ApiError> {
-    let search_request = grpc::SearchRequest {
+    let search_request = SearchRequest {
```

Comment:
I recommend keeping the `grpc::` in front of all protobuf generated struct.

Reply:
Thanks. I'll fix it.

Reply:
Please see https://github.com/quickwit-inc/quickwit/pull/163/commits/5d9c0d1d0ae6abd74312bca1082d845db4b201a2